src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
 
     def normalize(self, archive, logger) -> None:
         super().normalize(archive, logger)
+
+####################################################
+# Abstract energy classes
+####################################################
+
+class Energy(PhysicalProperty):
+    """
+    Abstract physical property section describing some energy of a (sub)system.
+    """
+
+    type = Quantity(
+        type=MEnum('classical', 'quantum'),
+        description="""
+        Refers to the method used for calculating the energy.
+
+        Allowed values are:
+
+        | Energy Type            | Description                               |
+
+        | ---------------------- | ----------------------------------------- |
+
+        | `"classical"`          |  The energy is determined via a classical mechanics formalism. |
+
+        | `"quantum"`            |  The energy is determined via a quantum mechanics formalism. |
+        """,
+    )
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule',
+        description="""
+        The value of the energy.
+        """,
+    )
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ClassicalEnergy(Energy):
+    """
+    Abstract physical property section describing some classical energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+        if not self.type:
+            self.type == 'classical'
+        elif self.type != 'classical':
+            logger.error(
+                f"Misidentified type for classical energy."
+            )
+
+class QuantumEnergy(Energy):
+    """
+    Abstract physical property section describing some quantum energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+        if not self.type:
+            self.type == 'quantum'
+        elif self.type != 'quantum':
+            logger.error(
+                f"Misidentified type for quantum energy."
+            )
```

Comment:
I definitely understand your point. My reasons for proposing to distinguish between the two were:

1. To help clarify the origin of certain energy contributions in the case of a hybrid model. E.g., you may have a QM/MM model and want to store a particular contribution to the total energy from the QM and MM parts of your model separately.
    - I will argue against myself here and say that the proper way to do this would be with the method_ref, which is true, but this is a bit opaque for a user. Having the `type` quantity makes it easier to see quickly what level of model one used to get this output.
2. There are certain energy contributions, e.g., kinetic energy, where the standard method of calculation is fundamentally different from a classical or a quantum model.
    - Again maybe it is not a good idea to conflate the output and the method too much. In fact this goes against a bunch of my comments where I asked if we should take the reference to a specific calculation method out of the description. So, from this perspective I am on the fence.
3. There are many examples where there is not a quantum counterpart to the classical energy. In addition to a few quantum phenomena as you mentioned, in classical simulations it is common to store the contribution to the total energy (usually per particle) from a particular term of the force field. 
    - E.g., you may want to know the total energy on a particle in a configuration coming from some dihedral potential only. 
    - In the previous schema it became unclear at times the usage of certain contributions. This is partially due to the inclusion of specific methodology for calculation within the description, but also because I think in some cases people had a certain class of models in mind. So, I was trying to prevent this a little. 
    - I think this is why you thought there is a confusion between energy and potential. The potential is indeed stored within the FF section of method. But the corresponding energy is the evaluation of that potential or set of potentials for a particular configuration.


All that being said, I definitely am not set on this distinction. Curious to hear your rebuttal given the additional information, and also the thoughts of @Bernadette-Mohr and @JosePizarro3

Reply:
Regarding your points:

1. QM/MM are indeed a mapping between methods and structure. How would you, using the distinction above, distinguish the energy contributions in QM/QM/MM (e.g. CCSD/DFT/Amber) or QM/MM/MM?
2. ... QM in the limit of heavy particles converges to CM, **without exception**. The main difference in maths is that variables and functions in CM become functions (in the Hilbert space) and operators in QM. That is to say, the same physics is presented in higher-order structures. Some new physical phenomena emerge, but all the old ones remain. More importantly, **why does the mathematical representation matter here?** We're just getting scalars (maybe tensors) in our data.
3. Well, that's just plain wrong. At best, these communities are not interested in these contributions, but that's another story. As for your example: the dihedral potential is captured by DFT, just not named as such. ML potentials actually use DFT data to reformulate the physics in terms of the (local) neighborhood.

I agree with your sentiment that a lot of these contributions are model-specific, and there's value in extracting those. Then we need a system to annotate the method terms. I would mentally distinguish it from the contributions of physical phenomena, which are not subordinate to the method used to simulate them. Let's reserve the energies here for what experimentalists could also request.

Reply:
Ok, here's my unqualified opinion:

1. Making the different energy contributions storable/accessible is valid, in my opinion, either because they were relevant for the original study or because somebody is interested in that particular aspect when conducting secondary analysis.
2. While I agree with @ndaelman-hu that, theoretically, especially QM  and MM/MD describe the same physics, each approach has it's strengths and weaknesses.
- We should absolutely provide that information somewhere, but maybe more with methods than with outputs?
One use case that comes to mind is training data for an ML model. If you want to have any faith in your predictions, you should be interested in the underlying distribution or potential issues.
3. @JFRudzinski, I'm not sure why you think including the methodology makes things confusing here; streamlining what information we provide where will probably make things clearer, though.

Generally, it seems that the slightly different usage of "potential" vs. "energy" in the different methods is a source of confusion even among us, so I think we should try to prevent this with the users. I remember us discussing something like a "derived" flag, to differentiate quantities that are calculated and aspects that are directly part of or automatically generated by the method. Could something along this line help?